VAMobile/src/utils/hooks/index.tsx

```diff
@@ -306,9 +306,17 @@ export function useDestructiveActionSheet(): (props: useDestructiveActionSheetPr
     showActionSheetWithOptions(
       {
         title: props.title,
-        titleTextStyle: { fontWeight: 'bold', textAlign: 'center', color: currentTheme.colors.text.primary },
+        titleTextStyle: {
+          fontWeight: 'bold',
+          textAlign: isIOS() ? 'center' : 'left',
```

Comment:
Instead of performing the check multiple times, can we have a function or a constant to ensure consistency?